site/content/blog/2020-04-17-zap-jwt-scanner/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "ZAP JWT Scanner"
+description: "How to scan JWT endpoints using OWASP ZAP"
+images:
+- https://www.zaproxy.org/blog/2020-04-17-zap-jwt-scanner/images/jwt-settings.png
+type: post
+tags:
+- blog
+date: "2020-04-17"
+addSocialPreview: true
+authors:
+    - preetkaran20 (Karan Preet Singh Sasan)
+---
+
+With the popularity of JSON Web Tokens there comes the need to secure them so that they are not misused because of the bad configuration or older libraries or buggy implementations. So JWT Scanner Addon is used to find such vulnerabilities and this blog explains on how to use JWT Scanner Addon.
+
+## Configuration
+As JWT Scanner runs with Active Scanner so most of the configurations are same as Active Scanner but there are certain configurations which are specific for JWT Scanner.
+Under Options tab you will find JWT settings section as shown below:
+[![JWT-Settings](./images/jwt-settings.png)]
+
+### Explanation
+In case the application which you are trying to scan is using RSA or more specifically RS* algorithm then please mention the public certificate TrustStore path and TrustStore password. These fields are used to find certain vulnerabilities related to RS* based JWT's.
```

Comment:
```suggestion
In case the application which you are trying to scan is using RSA or more specifically RS* algorithm then please configure the public certificate TrustStore path and TrustStore password. These fields are used to find certain vulnerabilities related to RS* based JWTs.
```

Public of private?